### ISSUE TITLE
Inference on MultiHMR using DUNE backbone

### ISSUE THREAD

**Issue by pradyumnaym** [naver/dune#3]
Hello, Thank you for sharing this great work!

I am trying to perform MultiHMR inference using the DUNE backbone, but am unable to reproduce MultiHMR quality results using DUNE. I assume I am doing something wrong when using DUNE for MultiHMR inference. Would it be possible to share a guide on how to use DUNE features with MultiHMR?

Using the DUNE backbone features, projecting using the provided projector, and de-normalizing before passing to MultiHMR does not seem to work for me.

It would also be great if the finetuned MultiHMR decoder can be released, since it seems a little sensitive to the change in input resolution.

Thank you!

**Comment by mbsariyildiz** [comment]
Hi @pradyumnaym , and thanks for using our model!

> Using the DUNE backbone features, projecting using the provided projector, and de-normalizing before passing to MultiHMR does not seem to work for me.

If I understand correctly, you tested the "plug-and-play" capabilities of the DUNE encoder and projector to replace the MultiHMR encoder. However this is not how we evaluate DUNE in the paper. After training the DUNE encoder, we freeze it and fine-tune the decoder (from the corresponding teacher) in each downstream task separately. See the Appendix B of the paper for defails. Sorry this was not mention in README.

Unfortunately we do not plan on releasing the fine-tuned MultiHMR decoder soon.

**Comment by pradyumnaym** [comment]
I see, thank you for the information!
